backend/main.py

```diff
@@ -14,13 +14,7 @@
 def startup_event():
     global conn, cursor
 
-    conn = psycopg2.connect(
-        database=os.getenv("database"),
-        host=os.getenv("db_host"),
-        user=os.getenv("db_user"),
-        password=os.getenv("db_password"),
-        port=os.getenv("db_port"),
-    )
+    conn = psycopg2.connect(url=os.getenv("DATABASE_URL"))
```

Comment:
This doesn't seem to work on my machine for whatever reason. Switching it back to manually specifying each component (with the new env variables) does work though.

Reply:
this is outdated anyway with #32, which changes to using sqlalchemy instead of psycopy2 directly.

can you try testing the connection with #32?